extension/intents/forms/formsContentScript.js

```diff
@@ -248,6 +244,51 @@ this.dictationContentScript = (function() {
     return `${startSlice}${spaceBefore}${snippet}${spaceAfter}${endSlice}`;
   }
 
+  function insertStringContentEditable(snippet, containerElement) {
+    if (window.getSelection) {
+        const sel = window.getSelection();
+        if (sel.getRangeAt && sel.rangeCount) {
```

Comment:
I think `sel.rangeCount` could be zero though, meaning there's no selection. That should probably default to appending then.

Reply:
Turns out `sel.rangeCount` is not needed as well as it is meant for multiple ranges/selections. The function only needs the cursor position which is provided by `sel.getRangeAt(0)`